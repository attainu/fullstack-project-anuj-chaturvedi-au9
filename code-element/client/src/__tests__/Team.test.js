
import React from 'react';
import { shallow } from 'enzyme';
import Team from '../components/pages/Team';
     
    it('testing Team page',()=>{
        const wrapper = shallow(<Team/>);
        expect(wrapper).toMatchSnapshot();
        
    });
    

