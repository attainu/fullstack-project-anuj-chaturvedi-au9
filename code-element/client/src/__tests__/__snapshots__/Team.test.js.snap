// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Team page 1`] = `ShallowWrapper {}`;
